modules/ROOT/pages/runtime-manager-agent-api.adoc

```diff
@@ -9,7 +9,7 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-For the vast majority of use cases, all you'll need to do with the Runtime Manager Agent is install it and configure it according to what you can read in xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring the Runtime Manager Agent]
+For the majority of use cases, all you'll need to do with the Runtime Manager Agent is install it and configure it according to what you can read in xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring the Runtime Manager Agent]
```

Comment:
suggest less awkward phrase than "according to what you can read in" (implies they may not be able to read), something like "install it, and then configure the agent using xref:installing-and-configuring-runtime-manager-agent.adoc[Installing and Configuring the Runtime Manager Agent].